.github/workflows/release-please.yml

```diff
@@ -19,5 +19,5 @@ jobs:
       - uses: googleapis/release-please-action@v4
         with:
             # Using a personal access token so releases created by this workflow can trigger the deployment workflow
-            token: ${{ secrets.RELEASE_PLEASE_PAT }}
```

Comment:
have these secrets been removed?

Reply:
That token was never in this repo, I added the workflow yesterday and was waiting for the bot to configure it.